Packages/Keystore/Sources/LocalKeystore.swift

```diff
@@ -141,12 +142,22 @@ public final class LocalKeystore: Keystore {
         try walletStore.nextWalletIndex()
     }
     
-    public func getPrivateKey(wallet: Wallet, chain: Chain) throws -> Data {
+    public func getPrivateKey(wallet: Primitives.Wallet, chain: Chain) throws -> Data {
         let password = try keystorePassword.getPassword()
         return try walletKeyStore.getPrivateKey(id: wallet.id, type: wallet.type, chain: chain, password: password)
     }
-    
-    public func getMnemonic(wallet: Wallet) throws -> [String] {
+
+    public func getPrivateKey(wallet: Primitives.Wallet, chain: Chain, encoding: EncodingType?) throws -> String {
```

Comment:
can you add tests? to cover base58 and hex

Reply:
done